hcn/hcnsupport.go

```diff
@@ -76,6 +80,15 @@ func GetSupportedFeatures() SupportedFeatures {
 	features.NetworkACL = isFeatureSupported(globals.Version, NetworkACLPolicyVersion)
 	features.NestedIpSet = isFeatureSupported(globals.Version, NestedIpSetVersion)
 
+	// Only print the HNS version and features supported once, instead of everytime this is invoked. These logs are useful to
+	// debug incidents where there's confusion on if a feature is supported on the host machine. The sync.Once helps to avoid redundant
+	// spam of these anytime a check needs to be made for if an HNS feature is supported. This is a common occurrence in kubeproxy
+	// for example.
+	versionOnce.Do(func() {
+		logrus.Infof("HNS version: %+v", globals)
```

Comment:
We should log using `WithField` instead of string interpolation.

Reply:
Sure, even with the supported features? That one's gonna get quite long haha

Reply:
Done

Reply:
I'm confused why we need the whole `featuresToFields` thing. Can't we just log the same values as before, but treat them as fields instead of interpolating?

e.g.
```
logrus.WithFields(logrus.Fields{
    "version":           fmt.Sprintf("%+v", globals.Version),
    "supportedFeatures": fmt.Sprintf("%+v", features),
}).Info("HCN feature check")
```

Reply:
Ah.. perhaps it's hit that hour where it's too late to be making code changes 😓